src/pretix/static/pretixpresale/scss/_event.scss

```diff
@@ -1,12 +1,22 @@
+.with-plus-circle-before {
+    position: relative;
+}
+.with-plus-circle-before:before {
+    display: block;
+    font-family: "FontAwesome";
+    content: "";
+    text-decoration: none;
+    position: absolute;
+    left: -1.2em;
+    top: 0.1em;
+}
```

Comment:
Since we dropped underlined links from this PR, we can also revert this hack probably?

In the specific case of the waiting list, I think a button could be fine as well. (Also, wouldn't be including a <span> element in the link and underlining only that possible much simpler to go forward? But we should discuss that in a new issue

Reply:
Removed that commit